content/post/2025-02-14-llm-codegen-and-you.md

```
@@ -0,0 +1,288 @@
+
+For this I have a slightly different method. It is similar to above, but a bit less “planning based.” The planning is done per task, not for the entire project.
+
+### Get context
+
+I think everyone who is knee deep in AI dev has a different tool for this, but you need something to grab your source code and efficiently jam it into the LLM.
+
+I currently use a tool called `[repomix](https://github.com/yamadashy/repomix)`. I have a task set system wide in my `.mise.toml` that allows me to do various things with my code base ([mise rules](https://mise.jdx.dev/)).
+
+Here is the llm task list:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)